example/.env

```diff
@@ -0,0 +1,4 @@
+ELK_VERSION=7.15.2
+CACHE_ROOT=data/logs
```

Comment:
CACHE_ROOT would be data, we should add the /logs within the script (or possibly on the next line for configurability
```
# Where is your cache root directory (By default this is the data folder within the main lancachenet docker-compose_
CACHE_ROOT=data
# And the logs (Default should be fine for 99% of cases)
CACHE_LOGS=${CACHE_ROOT}/logs
```
```

Reply:
Do we gain anything by having a separate env for cache logs? 

Would a rename of the env to `CACHE_LOGS_DIR` be sufficient here?

Reply:
people don't know where their logs are, they know where the cache root as, as thats whats defined in the main compose file. but you are actually after the logs. Hence it could be simplified to cache_Root and then add the /logs in the compose file/scripts